config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  root 'welcome#index'
+  # root 'welcome#index'
+  root to: 'landing_page#index'
```

Comment:
It's fine that you chose to do your own root path and landing page, but it also would've been fine just to use the welcome#index route and welcome controller and views that were provided to save time. Up to you, but it's not a requirement to create your own if not specified that you should.